timemachine/fe/bar.py

```diff
@@ -193,6 +189,9 @@ def bootstrap_bar(
     seed = 2022
     rng = np.random.default_rng(seed)
 
+    u_kn, N_k = ukln_to_ukn(u_kln)
+    mbar = pymbar.MBAR(u_kn, N_k, maximum_iterations=maximum_iterations)
```

Comment:
nit: this seems unnecessary vs. setting `initial_f_k=None` below (or maybe better, `initial_f_k=np.array([0.0, df])`)

Reply:
Good call, will replace it with `initial_f_k=np.array([0.0, df])` which is the same value without constructing an extra MBAR object

Reply:
Set using pre-existing value in https://github.com/proteneer/timemachine/pull/1344/commits/a7de23f8700f0af98ed72ea49ec221fdff85566f